<article class="bugdoc">
    <h2 class="bugdoc-title" id="Bug-Doc:PE22279">PE22279: S323</h2>
    <h3>S323</h3>
    <section id="tip">
     <h4 style="border-top: 0pt; padding: 0pt 4pt 4pt 4pt;">Bug info</h4>
                <dl>
                    <dt>BugTypeID</dt>
                    <dd>PE22279</dd>
                    <dt>Tags</dt>
                    <dd>flake8</dd>
                    <dt>Category</dt>
                    <dd>Code style and efficiency</dd>
                    <dt>Authors</dt>
                    <dd>pinpoint</dd>
                </dl>
    <h2>Reason</h2>

<pre><code>S323 By default, Python will create a secure, verified ssl context for use in such classes as HTTPSConnection. However, it still allows using an insecure context via the _create_unverified_context that reverts to the previous behavior that does not validate certificates or perform hostname checks.

</code></pre>
<h4>References</h4>
<section class="footnotes">
<ol>
<li id="fn-1"><p>https://www.python.org/dev/peps/pep-0008/<a href="#fnref-1" class="footnote">&#8617;</a></p></li>
</ol>
</section>

    </section>
</article>
